merlin/systems/dag/runtimes/triton/ops/operator.py

```diff
@@ -107,6 +67,7 @@ def transform(
         """
         return transformable
 
+    @abstractmethod
```

Comment:
This `export` method of the `TritonOperator` is now an abstractmethod. Meaning that an operator that inherits from TritonOperator is required to implement the export method.